plugin.video.retrospect/resources/libs/streams/mpd.py

```diff
@@ -46,6 +46,17 @@ def set_input_stream_addon_input(strm, proxy=None, headers=None,
 
         """
 
+        if license_key is not None:
```

Comment:
Same here, I'm not a fan of `is not None` in places where you can simply do `if something:`

Reply:
I understand. Is it OK if I pick with each piece of code I change for a next release. That way the code will increasingly become nicer.